config/201-channelable-manipulator-clusterrole.yaml

```diff
@@ -17,7 +17,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: events-channel-addressable-resolver
+  name: cloud-run-events-channel-manipulator
```

Comment:
This change seems irrelevant

Reply:
Yep. I think that's just a trivial renaming which is irrelevant to this ticket. I just followed the PR of Scott https://github.com/google/knative-gcp/pull/312/files to rename it.